go.mod

```diff
@@ -1,6 +1,8 @@
 module k8s.io/release
 
-go 1.21
+go 1.22.0
```

Comment:
@cpanato we reverted that in https://github.com/kubernetes/release/commit/6d2649680992d7309728e7ffd233ce33b6b98bda, can we now move over to go 1.22?

Reply:
we cannot due to https://github.com/kubernetes/release/blob/master/images/releng/k8s-ci-builder/variants.yaml